src/xenia/ui/window_gtk.cc

```diff
@@ -350,18 +405,20 @@ bool GTKWindow::HandleKeyboard(GdkEventKey* event) {
   bool ctrl_pressed = modifiers & GDK_CONTROL_MASK;
   bool alt_pressed = modifiers & GDK_META_MASK;
   bool super_pressed = modifiers & GDK_SUPER_MASK;
+  uint32_t key_char = gdk_keyval_to_unicode(event->keyval);
```

Comment:
Is this unused, except for the `> 0` check? I think it should be written in the `KeyEvent` somewhere, or is requested in some other place too?